packages/dashboard/theme/lumo/vaadin-dashboard-widget-styles.js

```diff
@@ -0,0 +1,230 @@
+import '@vaadin/vaadin-lumo-styles/color.js';
+import '@vaadin/vaadin-lumo-styles/sizing.js';
+import '@vaadin/vaadin-lumo-styles/spacing.js';
+import '@vaadin/vaadin-lumo-styles/style.js';
+import '@vaadin/vaadin-lumo-styles/typography.js';
+import '@vaadin/vaadin-lumo-styles/font-icons.js';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+const dashboardWidgetAndSection = css`
+  :host {
+    border-radius: var(--lumo-border-radius-l);
+    color: var(--lumo-body-text-color);
+    font-family: var(--lumo-font-family);
+    font-size: var(--lumo-font-size-m);
+    line-height: var(--lumo-line-height-m);
+    --_focus-ring-color: var(--vaadin-focus-ring-color, var(--lumo-primary-color-50pct));
+    --_focus-ring-width: var(--vaadin-focus-ring-width, 2px);
+    --_focus-ring-spacing-max-offset: 0px;
+    /* Calculates the offset by which the focus ring should be shifted inwards based on a custom --vaadin-dashboard-spacing value.
+    Effectively keeps the focus ring visible if --vaadin-dashboard-spacing is set to 0px */
+    --_focus-ring-spacing-offset: min(
+      max(calc(var(--_focus-ring-width) * -1), var(--_vaadin-dashboard-spacing) - var(--_focus-ring-width)),
+      var(--_focus-ring-spacing-max-offset, 0px)
+    );
+    outline: none;
+  }
+
+  :host([focused])::before {
+    content: '';
+    display: block;
+    position: absolute;
+    inset: 0;
+    border-radius: var(--lumo-border-radius-l);
+    pointer-events: none;
+    margin: calc(var(--_focus-ring-spacing-offset) * -1);
+    outline: var(--_focus-ring-width) solid var(--_focus-ring-color);
+  }
+
+  :host([selected])::before {
+    box-shadow:
+      0 2px 4px -1px var(--lumo-primary-color-10pct),
+      0 3px 12px -1px var(--lumo-primary-color-50pct);
+  }
+
+  /* Buttons styling */
+  vaadin-button {
+    font-family: 'lumo-icons';
+    font-size: var(--lumo-icon-size-s);
+  }
+
+  .icon::before {
+    display: block;
+    content: var(--icon);
+  }
+
+  :host(:not([selected])) *:not(.mode-controls) vaadin-button,
+  :host([move-mode]) *:not(.mode-controls) vaadin-button,
+  :host([resize-mode]) *:not(.mode-controls) vaadin-button {
+    color: var(--lumo-disabled-text-color);
+  }
+
+  /* Header styling */
+  header {
+    display: flex;
+    align-items: center;
+    padding: var(--lumo-space-s) var(--lumo-space-m);
+    gap: var(--lumo-space-s);
+  }
+
+  :host([editable]) header {
+    padding-inline: var(--lumo-space-s);
+  }
+
+  /* Drag handle styling */
+  #drag-handle {
+    cursor: move;
+    --icon: var(--lumo-icons-menu);
+  }
+
+  /* Remove button styling */
+  #remove-button {
+    cursor: pointer;
+    --icon: var(--lumo-icons-cross);
+  }
+
+  /* Title slot styling */
+  slot[name='title']::slotted(*) {
+    flex: 1;
+    font-size: var(--lumo-font-size-m) !important;
+    font-weight: 500;
+    color: var(--lumo-header-text-color) !important;
```

Comment:
Using `!important` here is a temporary workaround. We're considering moving the title element inside the component's shadow root instead.